src/wasm-interpreter.h

```diff
@@ -1858,8 +1872,8 @@ class ConstantExpressionRunner : public ExpressionRunner<SubType> {
   Flow visitGlobalGet(GlobalGet* curr) {
     NOTE_ENTER("GlobalGet");
     NOTE_NAME(curr->name);
-    if (module != nullptr) {
-      auto* global = module->getGlobal(curr->name);
+    if (this->module != nullptr) {
```

Comment:
Why is the `this->` necessary?

Reply:
It's the C++ thing where one template is the subclass of another template. It won't compile without this, as it says "`'module' was not declared in this scope`.

Reply:
Yuck 😷 Thanks for the explanation!